src/exercises/day-1/luhn.rs

```diff
@@ -15,34 +15,38 @@
 // ANCHOR: solution
 // ANCHOR: luhn
 pub fn luhn(cc_number: &str) -> bool {
-    // ANCHOR_END: luhn
```

Comment:
The comment here is important: it is used to include just the function signature into the exercise slide.